### DISCUSSION TITLE
Call for solidarity

### DISCUSSION THREAD

**Discussion by philippoo66** [Andre0512/hon#150] [General]
### Call for solidarity and the preservation of open source solutions that meet user needs!

**I find it alarming that corporations want to suppress the development of free software!**

Of course, no independent developer wants to get involved in a legal dispute with a corporation, so **we all** should solve the matter differently! 

**I call on everyone who has a github account to fork the threatened projects!** 

If the matter spreads faster than Haier can send warnings, the open source idea will be saved. I have already forked both projects, as have over 800 others. As soon as someone receives a takedown notice, it will of course be followed, but by then there will hopefully be dozens more forks.

It's about hon and pyhon. Please uncheck “_Fork Main Brunch only_” so that the entire project is forked. And please tell your friends too! Even if you don't have a Haier device right now - **We are the people! and we shouldn't let corporations ban us from open source thinking!**

thank you & greetings in solidarity!
Phil

**Comment by GeneralGeeCo** [comment]
https://github.com/Andre0512/pyhOn

**Reply by philippoo66** [reply]
we need to find a way to always identify a one 'official' fork that Andre continues to maintain. and when this will receive the takedown note, the next one has to get the 'official' one...

we need to do so until fu.. Haier will realize that open source thinking is superior and get tired sending takedown notes...

**Reply by philippoo66** [reply]
Perhaps it's also a good idea that all owners (and also all 'possible' owners) of Haier devices schould send an email to Haier and ask them to cancel that takedown note!?
